src/movies.js

```diff
@@ -1,16 +1,113 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies){
+
+const directorF =  movies.map(function(movie){
+        return movie.director
+    })
+
+return directorF
+}
+
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(movies){
+    let drama;
+    let spiel = movies.filter((movie) => {
+    for (let i=0; i < movie.genre.length; i++){
+       drama = movie.genre[i] == "Drama" && movie.director == "Steven Spielberg";
+       if(drama){break;}
+    } 
+    return drama
+}) 
+return spiel.length;
+};
+
+function howManyMovies(movies){
```

Comment:
This is another function with the same name as before, which is causing issues. It will only run the second one, and in this case, this one is not checking for the director. If you comment or remove it the tests for the howManyMovies function will pass.